libs/core/langchain_core/load/serializable.py

```diff
@@ -186,6 +187,23 @@ def to_json_not_implemented(self) -> SerializedNotImplemented:
         return to_json_not_implemented(self)
 
 
+def _is_field_useful(inst: Serializable, key: str, value: Any) -> bool:
+    """Check if a field is useful as a constructor argument.
+
+    Args:
+        inst: The instance.
+        key: The key.
+        value: The value.
+
+    Returns:
+        Whether the field is useful.
+    """
+    field = inst.__fields__.get(key)
+    if not field:
+        return False
+    return field.required is True or field.get_default() != value
```

Comment:
Hmmm for the second condition, if we do something like:
```
ChatOpenAI(model="gpt-3.5-turbo").to_json()
```
the model name is elided.

We really shouldn't have put default model names in these classes, but now that that bridge has been crossed it seems not great to then lose that info in the serialized output